docker/backend/mempool-config.json

```diff
@@ -67,6 +67,22 @@
     "ENABLED": __BISQ_ENABLED__,
     "DATA_PATH": "__BISQ_DATA_PATH__"
   },
+  "LIGHTNING": {
+    "ENABLED": __LIGHTNING_ENABLED__,
+    "BACKEND": "__LIGHTNING_BACKEND__",
+    "STATS_REFRESH_INTERVAL": __LIGHTNING_STATS_REFRESH_INTERVAL__,
+    "GRAPH_REFRESH_INTERVAL": __LIGHTNING_GRAPH_REFRESH_INTERVAL__,
+    "LOGGER_UPDATE_INTERVAL": __LIGHTNING_LOGGER_UPDATE_INTERVAL__,
+    "TOPOLOGY_FOLDER": "__LIGHTNING_TOPOLOGY_FOLDER__"
+  },
+  "LND": {
+    "TLS_CERT_PATH": "__LND_TLS_CERT_PATH__",
+    "MACAROON_PATH": "__LND_MACAROON_PATH__",
+    "REST_API_URL": "__LND_REST_API_URL__"
+  },
+  "CLIGHTNING": {
+    "SOCKET": "__CLN_SOCKET__"
```

Comment:
```suggestion
    "SOCKET": "__CLIGHTNING_SOCKET__"
```

Reply:
`__CLN_SOCKET__` is defined in start.sh which has been approved before. Your suggestion would require a change in start.sh as well, is that preferred?

https://github.com/mempool/mempool/blob/master/docker/backend/start.sh#L108

Reply:
Yes. `__CLN_SOCKET__`  was a mistake because the key in the config.json file is under `CLIGHTNING`. Ideally we should keep all the keys in sync. There is one exception (`BACKEND_HTTP_PORT`) that I'm going to fix before we tag the next release.